recipes-bsp/barebox/barebox.inc

```diff
@@ -61,6 +61,8 @@ do_configure() {
 	cml1_do_configure
 }
 
+BAREBOX_FIRMWARE_DIR ?= ""
```

Comment:
please add a `BAREBOX_FIRMWARE_DIR[doc]` entry shortly describing the usage and purpose of this variable as this is possibly not straightforward.

Reply:
Done.